astropy/units/quantity.py

```diff
@@ -39,6 +39,15 @@
 _UNIT_NOT_INITIALISED = "(Unit not initialised)"
 _UFUNCS_FILTER_WARNINGS = {np.arcsin, np.arccos, np.arccosh, np.arctanh}
 
+def quantity_allclose(a, b, rtol=1.e-5, atol=None, **kwargs):
```

Comment:
I would vote for this just being called `allclose` as it's in `units.quantity`

Reply:
Agreed. It should also be added to `__all__`, so that it is exposed as `u.allclose()`.

Aside: needs an extra empty line above the function definition (pep8)

Reply:
After going through `units.quantity` all the attributes of __all__ .All of them are classes which have functions inside them , so i have to do the same?

Reply:
No, this would remain a function - it is just a coincidence that in this module only classes are exported.